scripts/components/OpenSearch-Dashboards/install.sh

```diff
@@ -80,9 +80,8 @@ cd $DIR
 ## Setup default config
 if [ "$DISTRIBUTION" = "tar" ]; then
     cp ../../../config/opensearch_dashboards.yml "$OUTPUT/config/"
-
-elif [ "$DISTRIBUTION" = "rpm" ]; then
+elif [ "$DISTRIBUTION" = "rpm" -o "$DISTRIBUTION" = "deb" ]; then
```

Comment:
`-o`? Should this be some kind of `||`?

Reply:
It's using Bash https://github.com/opensearch-project/opensearch-build/pull/2526/files/6bdba5d1e4480bc85c45650d4573f042904c3e3a#diff-678e1fc6dbb729b111525a80594ce6d02a00635590b66d161f21982897653d31R1

```shell
if [ EXPR1 -o EXPR2 ]
  then
    echo "Hello World"
fi
```

`True if either EXPR1 or EXPR2 is true.`

https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html#tab_07_02